### ISSUE TITLE
Support for Webpack 4?

### ISSUE THREAD

**Issue by Nickersoft** [negibouze/html-webpack-pug-plugin#15]
Are there any plans to adapt this plugin for Webpack 4? I could try my hand at a pull request if you're too busy. Currently using this plugin with Webpack 4 renders the following error:

```
ERROR in   TypeError: callback is not a function
      
      - index.js:65 HtmlWebpackPugPlugin.preProcessHtml
        [homepage]/[html-webpack-pug-plugin]/index.js:65:3
      
      - index.js:14 
        [homepage]/[html-webpack-pug-plugin]/index.js:14:12
      
      
      - new Promise
      
      
      - Hook.js:35 AsyncSeriesWaterfallHook.lazyCompileHook [as _promise]
        [homepage]/[tapable]/lib/Hook.js:35:21
      
      - index.js:645 
        [homepage]/[html-webpack-plugin]/index.js:645:47
      
      - index.js:173 Promise.resolve.then.then.then.then.then.then.html
        [homepage]/[html-webpack-plugin]/index.js:173:18
      
      
      - next_tick.js:160 process._tickCallback
        internal/process/next_tick.js:160:7
```

**Comment by Nickersoft** [comment]
Went ahead and made a PR: https://github.com/negibouze/html-webpack-pug-plugin/pull/16

**Comment by negibouze** [comment]
@Nickersoft Thank you for your Report & PR.

**Comment by negibouze** [comment]
@Nickersoft 
I released a new version(v0.3.0).
Thank you for your cooperation :)
